python/ucxx/_lib/tests/conftest.py

```diff
@@ -14,6 +14,6 @@
 os.environ["RAPIDS_NO_INITIALIZE"] = "True"
 
 
-@pytest.hookimpl(hookwrapper=True)
+@pytest.hookimpl()
```

Comment:
I'm not sure you need this decorator at all?

Reply:
`@pytest.hookimpl()` is required otherwise it won't apply to tests, not automatically at least.

Reply:
Hmm, I was confused then. If I have:

```
# conftest.py
def pytest_runtest_teardown(item, nextitem):
    print("Running teardown")
```

```
# test.py

def test():
    print("Running test")
```

Then
```
pytest test.py -s -v
[...]
test.py::test Running test
PASSEDRunning teardown
```

The addition of the decorator doesn't make a difference.

I _think_ this is because the name `pytest_runtest_teardown` is already registered as a hook name, so when pytest collects the conftest.py it notices the hook and adds it to the known hooks to be called.

Reply:
Ok, I must have done something wrong then. I initially tested without the decorator and for some reason it didn't print, I tried this again now and it prints. Removed it now in https://github.com/rapidsai/ucxx/pull/139/commits/c79bd35a37aaaf68eb5a8de42fa63e3e26dd7c28 .